src/lib/conversion/rotation.h

```diff
@@ -91,8 +91,10 @@ enum Rotation : uint8_t {
 	ROTATION_ROLL_90_PITCH_68_YAW_293 = 38,
 	ROTATION_PITCH_315                = 39,
 	ROTATION_ROLL_90_PITCH_315        = 40,
+	ROTATION_MAX, // NOTE: Update this variable to match the number of non-custom Rotation enum members!
```

Comment:
```suggestion
	ROTATION_MAX,
```
This is a good example to show that comments rot quickly; even in the same PR they were created.

Reply:
Oh yes, good to remove the comment, but I meant that it should be placed at the end of the 'non-custom' rotation (in case someone would place it after the ROTATION_CUSTOM :exploding_head:)

Reply:
I wanted to emphasize that this shouldn't be placed after the ROTATION_CUSTOM. Would this comment not be necessary?